packages/slice-machine/src/components/Card/Card.module.css

```diff
@@ -0,0 +1,185 @@
+.flex {
+  all: unset;
+  display: flex;
+}
+
+.grid {
+  all: unset;
+  display: grid;
+}
+
+.column {
+  composes: flex;
+  flex-direction: column;
+}
+
+.root {
+  composes: column;
+  position: relative;
+  box-sizing: border-box;
+  border-color: var(--grey6);
+  border-radius: 6px;
+  border-style: solid;
+  border-width: 1px;
+  overflow-x: hidden;
+}
+
+.size-small {
+  height: 240px;
+}
+
+.size-medium {
+  height: 320px;
+}
+
+.solid {
+  background-color: var(--grey3);
+}
+.solid[data-state="checked"] {
+  border-color: var(--purple10);
+}
+
+.outlined {
+  background-color: var(--grey2);
+}
+.outlined[data-state="checked"] {
+  border-color: var(--purple8);
+}
+
+.interactive:focus-visible {
+  box-shadow: var(--box-shadow-purple-focus);
+}
+.interactive[data-disabled] {
+  opacity: 0.5;
+}
+.interactive:not([data-disabled]) {
+  cursor: pointer;
+}
+
+.interactive.solid:focus-visible {
```

Comment:
💡 General selectors are generally preferred over specific ones since they avoid specificity issues. In this case, `.interactive.solid` is more specific than something like `.interactive-solid`.

Before:
```css
.interactive.solid {
  /* Your interactive-solid variant styles... */
}
```

After:
```css
.interactiveBase {
  /* Your interactive styles... */
}

.interactive-solid {
  composes: interactiveBase;
  /* Your interactive-solid variant styles... */
}
```

This change would require adding/restoring an entry to `clsx`.

I'll let you decide which one you prefer since I can see this component is fairly complex. Chaining the class names like you have now might be simpler than what I am suggesting.

Reply:
I'll leave this with the chaining as I did it. I see and understand your point and will do it that way on other componentns. For this one, though, since it's complex, I think the chaining makes the styling simpler.